Packages/manifest.json

```diff
@@ -1,14 +1,31 @@
 {
   "dependencies": {
-    "com.unity.analytics": "exclude",
-    "com.unity.ads": "exclude",
-    "com.unity.standardevents": "exclude",
-    "com.unity.purchasing": "exclude",
     "com.unity.package-manager-ui": "1.9.11",
-    "com.unity.textmeshpro": "exclude",
-    "com.unity.modules.unityanalytics": "exclude",
-    "com.unity.modules.ai": "exclude",
-    "com.unity.modules.terrain": "exclude",
-    "com.unity.modules.terrainphysics": "exclude"
+    "com.unity.modules.animation": "1.0.0",
```

Comment:
revert this change

Reply:
* revert all project config changes

Reply:
https://github.com/vimeo/vimeo-unity-sdk/pull/31/commits/fac6d6eb15006e8e6bd457e255611f5c03bec544